public/controllers/agent/components/agents-preview.js

```diff
@@ -93,7 +94,12 @@ export const AgentsPreview = compose(
       return prev;
     }, {});
   };
-
+  async getManagers() {
+    const agents = await WzRequest.apiReq('GET', '/agents', {});
+    const managers = agents.data.data.affected_items.map((agent) => agent.manager);
+    const uniqueManagers = managers.reduce((unique,manager) => unique.includes(manager) ? unique : [...unique,manager], []);
+    return uniqueManagers;
+  }
```

Comment:
This could cause bugs due to:
- The request is limited to the first 500 agents. If we need more agents, we need to paginate. It costs so much to get the managers.
- This could work if we get all agents because could exist an agent registered with other managers that are not returned with the request.